pyemd/emd.pyx

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python3
```

Comment:
This line actually isn't useless; see http://stackoverflow.com/a/2429515/1085344.